ui/ui-components/components/export/List.tsx

```diff
@@ -174,6 +220,15 @@ export default function ExportsList(props: Props) {
                     <br />
                     <span>State</span>:{" "}
                     <StateBadge state={_export.state} marginBottom={0} />
+                    {["failed", "canceled"].includes(_export.state) ? (
+                      <FontAwesomeIcon
+                        className="ml-1"
+                        icon="sync-alt"
+                        size="xs"
+                        style={{ cursor: "pointer" }}
+                        onClick={() => retryExport(_export, index)}
+                      />
+                    ) : null}
```

Comment:
@bleonard If you get a chance to look over things, this is what this looks like.
![Screen Shot 2022-02-15 at 4 00 18 PM](https://user-images.githubusercontent.com/4967694/154148331-91c3b30c-63f5-4f9a-9b94-c392a4de2f21.png)